### ISSUE TITLE
Adding Robinhood exchange

### ISSUE THREAD

**Issue by nlapham21** [carlos8f/zenbot#1031]
Future enhancement question:

Any intention of adding Robinhood as an exchange so zenbot can be used to trade stocks available on their exchange as well?

**Comment by danielkhalilov** [comment]
Seems do-able, i think the steps to achieve this would be to pull historical stock/trade data from Robinhood into the Mongo database (retaining the same format as the crypto data goes in to feed into the models), then modify the code to API to Robinhood when executing buy/sell orders

**Comment by takiyon** [comment]
This would be awesome!!!

**Comment by markalinn** [comment]
Anyone take a look at this recently?

**Comment by BAKFR** [comment]
As far as I know, nobody is working on it. There is no active dev on Zenbot, only a few guys like me fixing the bugs that annoy us.

If you're a dev, feel free to propose a pull request.

**Comment by hrufine** [comment]
Yeah.I am planning to customize Gekko or zenbot for indian stock provider like robinhood.if anyone interested will proceed further, i am Node js developer,let me fork and see how complex it is,
